app/src/main/java/org/mozilla/fenix/library/downloads/DownloadFragment.kt

```diff
@@ -95,7 +161,64 @@ class DownloadFragment : LibraryPageFragment<DownloadItem>(), UserInteractionHan
         showToolbar(getString(R.string.library_downloads))
     }
 
+    override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
+        val menuRes = when (downloadStore.state.mode) {
+            DownloadFragmentState.Mode.Normal -> R.menu.library_menu
+            is DownloadFragmentState.Mode.Editing -> R.menu.download_select_multi
+        }
+        inflater.inflate(menuRes, menu)
+
+        menu.findItem(R.id.delete_downloads_multi_select)?.title =
+            SpannableString(getString(R.string.bookmark_menu_delete_button)).apply {
```

Comment:
No reason except trying to keep parity with the HistoryFragment that this was taken from, should I change it in both?

Reply:
It really depends if `HistoryFragment`has a strong reason for having it there, if it does I will suggest keep it there and remove it here as we don't need it here, right?

Reply:
Because of this error: None of the following candidates is applicable because of receiver type mismatch:
public fun SpannableString.setTextColor(context: Context, colorResId: Int): Unit defined in org.mozilla.fenix.ext in file SpannableString.kt